core/src/lib.rs

```diff
@@ -170,25 +230,33 @@ pub struct InputState {
     pub play: bool,
 }
 
+// A game from the perspective of a specific player
+pub struct GameAsPlayer {
+    my_uuid: UUID,
+    inputs: InputState,
+    pub game: Game,
+    socket: Box<dyn GameTx>,
+}
+
+// The full game state
+#[derive(PartialEq, Clone, Debug)]
 pub struct Game {
```

Comment:
Suggestion: s/Game/GameState. This would make the comment above obsolete and avoid "game.game" references.

Reply:
+1, I like it!

Done